File: tournaments/threeAndFour/threeAndFour.cs
int[] threeAndFour(int n) {
    var result = new List<int> {0};
    var item = 12;
    
    while (item < n) {
        if (0 == item % 3 && 0 == item % 4) {
            result.Add(item);
        }
        ++item;
    }
    
    return result.ToArray();
}

File: tournaments/threeAndFour/threeAndFour.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub threeAndFour {
    my ($n) = @_;

    my @result = (0);
    my $item = 12;

    while ($item < $n) {
        if (!($item % 3) && !($item % 4)) {
            push(@result, $item);
        }
        ++$item;
    }

    return \@result;
}
